docs/sample.wkt.csv

```diff
@@ -0,0 +1,4 @@
+#WKT,tag
```

Comment:
Maybe change this first line to:
# WKT, region
to better align with the documentation. (I had used tag when I made the file initially, but I think region is a good name)